pyproject.toml

```
@@ -28,20 +28,23 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = ">=3.8,<3.11"
-clinica = "^0.7.4"
+clinica = "^0.7.6"
 torch = "^1.8.0"
 torchvision = "*"
 tensorboard = "*"
 toml = "*"
 pandas = "^1.2"
 numpy = "^1.17"
 scikit-learn = "^1.0"
-scikit-image = "^0.19"
+scikit-image = ">=0.21"
```

I'd advise keeping the caret operator. There is no guarantee the API won't be changed in a backward incompatible way in the future.

```suggestion
scikit-image = "^0.21"
```